web/crux/src/app/config.bundle/config.bundle.message.ts

```diff
@@ -0,0 +1,15 @@
+import { UniqueKeyValue } from 'src/domain/container'
+
+export const WS_TYPE_PATCH_CONFIG_BUNDLE = 'patch-config-bundle'
+export type PatchConfigBundleEnvMessage = {
+  name?: string
+  environment?: UniqueKeyValue[]
+}
+
+export const WS_TYPE_CONFIG_BUNDLE_UPDATED = 'config-bundle-updated'
+export type ConfigBundleEnvUpdatedMessage = {
```

Comment:
Regarding that the 2 object content is the same, it would be better to use 1 outsourced, no? 🤔

Reply:
We have separate types for each message type even if they are the same, so I just wanted to follow how we do it generally.

Also I think having different types is beneficial, because the type name suggests what it does, instead of having something like `DeploymentEnvironmentMessage`.

Reply:
Okey, it was just a question, resolving the comment.